src/service/AppealStorageService.ts

```diff
@@ -0,0 +1,41 @@
+import axios, { AxiosRequestConfig, AxiosResponse } from 'axios';
+import { CREATED } from 'http-status-codes';
+
+import { Appeal } from 'app/models/Appeal';
+
+export class AppealStorageService {
+
+    constructor(private readonly uri: string) {
+        this.uri = uri;
+    }
+
+    public async save(appeal: Appeal, token: string): Promise<any> {
```

Comment:
Please change return type to `Promise<string>`

Reply:
done